templates/registration/login.html

```diff
@@ -10,7 +10,10 @@
   <meta http-equiv="x-ua-compatible" content="ie=edge">
 
   <title>{{ school_name | default:"MySchool" }}</title>
-
+  <!--Boot strap -->
+  <link href="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
```

Comment:
can you include the *.min.css file into static directory.

If user's internet isn't available, then loading of bootstrap link gets failed. (thinking loud).

Reply:
For that i think whole bootstrap code need to be place inside that *.min.css

Reply:
this project is fully based on [adminlte template ](https://adminlte.io/). The base stylesheet is packed with bootstrap. Can you use that instead or you have a reason to have to specifically load bootstrap?